server/lib/validation/languages/yue.js

```diff
@@ -25,8 +32,16 @@ const INVALIDATIONS = [{
   regex: /(\u00a9|\u00ae|[\u2000-\u3300]|[\u2580-\u27bf]|\ud83c[\ud000-\udfff]|\ud83d[\ud000-\udfff]|\ud83e[\ud000-\udfff]|[\ue000-\uf8ff])/,
   error: "句子唔可以含有 emoji 或者其他特殊 Unicode 符號",
 }, {
+  // No Mandarin sentence final particles
+  // 吧唄吶
   regex: /[\u5427\u5504\u5436](\s|\u3002|\u002E|\uFF0C|\u002C|$)/,
-  error: '句子唔可以有官話語氣詞（例如吧、唄、吶）',
+  error: '句子唔可以有官話語氣詞（吧唄吶）',
+},
+{
+  // No Mandarin characters
+  // 這哪您們唄咱啥甭
+  regex: /[\u9019\u54EA\u60A8\u5011\u54B1\u5565\u752D]/,
+  error: '句子唔可以有官話詞（這哪您們咱啥甭）',
```

Comment:
Are these the most common characters?

Reply:
Yes, this is not an exhaustive list of Mandarin-unique characters, but the most common ones.

Reply:
Ok, great. Thanks!